static/js/components/UserChip_test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+
+import { USER_PROFILE_RESPONSE } from '../constants';
+import R from 'ramda';
+import UserChip from './UserChip';
+import { getPreferredName } from '../util/util';
+import ProfileImage from './ProfileImage';
+
+describe('UserChip', () => {
+  let clone, chip;
+
+  const renderChip = profile => (
+    chip = shallow(<UserChip profile={profile} />)
+  );
+
+  beforeEach(() => {
+    clone = R.clone(USER_PROFILE_RESPONSE);
+  });
+  afterEach(() => chip = undefined);
+
+  it('should show the preferred name', () => {
+    renderChip(clone);
```

Comment:
i think it's preferable to define `chip` inside each of these functions (`const chip = renderChip(clone);`) instead of relying on a side effect and cleaning up that side effect in the `afterEach` method